src/components/forms/MultilineTextField.tsx

```diff
@@ -166,7 +166,7 @@ export default function MultilineTextField({
               {`${value.length} / ${characterLimit}`}
             </p>
           )}
-          <div css={{ paddingTop: 3 }}>
+          <div css={{ paddingTop: 3, marginLeft: characterLimit ? undefined : 'auto' }}>
```

Comment:
**question**: Not an experienced Emotion 👨‍🎤 user. does `marginLeft: undefined` boil down into: `margin-left: unset`?

Reply:
I _**think**_ from my past memory of checking this behavior in the dev tools, the styling just gets ignored when set to undefined. If memory serves me, the inline styling would show as `padding-top: 3; margin-left: auto` if there is not a character limit and `padding-top: 3` if there is a character limit.

Reply:
👍 🙇